### ISSUE TITLE
BS_MAXIMIZE damage variance

### ISSUE THREAD

**Issue by dragonis1701** [rathena/rathena#1396]
Damage should be the max possible with no variance but that is not the case. The damage does not change when using BS_MAXIMIZE.

**Comment by Playtester** [comment]
Pre-renewal or renewal?

**Comment by dragonis1701** [comment]
Pre-renewal.

**Comment by dragonis1701** [comment]
I figured out what it was, it was 
vit_def = def2*(def2-15)/150;
vit_def = def2/2 + (vit_def>0?rnd()%vit_def:0);
and
vit_def = (def2/20)*(def2/20);
vit_def = def2 + (vit_def>0?rnd()%vit_def:0);

I changed it to:
if (sc && sc->data[SC_MAXIMIZEPOWER]) {
			vit_def = def2*(def2-15)/150;
			vit_def = def2/2 + 0%vit_def;
		} else {
			vit_def = def2*(def2-15)/150;
			vit_def = def2/2 + (vit_def>0?rnd()%vit_def:0);
		}
and 
if (sc && sc->data[SC_MAXIMIZEPOWER]) {
			vit_def = (def2/20)*(def2/20);
			vit_def = def2 + 0%vit_def;
		} else {
			vit_def = (def2/20)*(def2/20);
			vit_def = def2 + (vit_def>0?rnd()%vit_def:0);
		}

No errors and it does correct damage with and without power maximize active.

**Comment by Playtester** [comment]
Power Maximize makes minimum vit def apply?

**Comment by dragonis1701** [comment]
Yeah, that was the youtube video I used as my source. Found it on a previous bug report on rAthena.

**Comment by Playtester** [comment]
I thought about this, but after some testing, the video is not a proof of this. Myst Case and Cruiser both don't have enough VIT to create any variance in the first place.
